dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/SendItemRequestAction.java

```diff
@@ -118,7 +118,11 @@ public Map act(Redirector redirector, SourceResolver resolver, Map objectModel,
                 )
                 .getRequestItemAuthor(context, item);
 
-        RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
+	// probably wrong -- Boolean.getBoolean() returns true if and only if
+	// the system property named by the argument exists and is equal to the
+	// string "true".
+        // RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
```

Comment:
Minor note... you can strip out all these comments, as they don't need to be in the code itself (and they just describe why you changed the below line).  

Instead, if you want to explain this difference, you can do so in the PR description/comments.